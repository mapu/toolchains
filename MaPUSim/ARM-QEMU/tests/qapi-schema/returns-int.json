# It is okay (although not extensible) to return a non-dictionary
# But to make it work, the name must be in a whitelist
{ 'command': 'guest-get-time', 'returns': 'int' }
