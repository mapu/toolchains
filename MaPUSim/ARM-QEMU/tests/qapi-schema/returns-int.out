[OrderedDict([('command', 'guest-get-time'), ('returns', 'int')])]
[]
[]
